rust/scx_loader/src/utils.rs

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-2.0
+//
+// Copyright (c) 2024 Vladislav Nepogodin <vnepogodin@cachyos.org>
+
+// This software may be used and distributed according to the terms of the
+// GNU General Public License version 2.
+
+use crate::SupportedSched;
+
+/// Get the scx name from the given scx trait
+pub fn get_name_from_scx(supported_sched: &SupportedSched) -> &'static str {
```

Comment:
silly question, could this function be an `impl` method on the `SupportedSched` enum?

Reply:
I think implementing Into/IntoStr would be more native